.gitignore

```diff
@@ -0,0 +1,3 @@
+Go/go.mod
```

Comment:
can you create a .gitignore in the Go folder versus updating the one at root? I need to move that one to /CSharp as its for .NET

Reply:
Done